packages/flutter/test/material/text_field_test.dart

```diff
@@ -1018,6 +1018,80 @@ void main() {
     expect(controller.text, 'abc d${testValue}ef ghi');
   });
 
+  testWidgets('Cannot interact when using TextField at the top of the screen',
```

Comment:
Is this description right? Shouldn't it be check that the toolbar appears below the textfield when there isn't enough space above the textfield to show it?